### ISSUE TITLE
ui: amend brief format for glossary terms

### ISSUE THREAD

**Issue by tiborsimko** [cernopendata/opendata.cern.ch#3345]
**Current behaviour**

On `master` branch, load glossary terms as follows:

```console
$ docker exec -i -t opendatacernch_web_1 cernopendata fixtures glossary --mode insert-or-replace
```

Then search in the web UI for LHCb glossary terms containing memdoc:

- http://localhost:5000/search?q=memdoc&l=list&order=asc&p=1&s=10&sort=bestmatch

![brief-master](https://user-images.githubusercontent.com/517546/218469686-06fafcc3-3093-4907-9336-d876fce27af8.png)

Compare to the same search on `qa` or `production` branches:

- https://opendata.cern.ch/search?page=1&size=20&q=memdoc

![brief-production](https://user-images.githubusercontent.com/517546/218469771-8e277aba-f49c-4bde-91f3-f9cc09b04752.png)

Basically, unlike on `qa` or `production`, the `master` branch does not "filter out" HTML in the brief formats of glossary records. This is a consequence of Angular -> React Semantic UI change.

**Expected behaviour**

Amend brief output format templates to filter away HTML in LHCb glossary records so that the output on `master` is nicely readable as on `qa` or `production`.
